activerecord/CHANGELOG.md

```diff
@@ -102,4 +102,8 @@
 
     *Kazuma Watanabe*
 
+*   Fix MySQL default functions getting dropped when changing a column's nullability.
+
+    *Bastian Bartmann*
```

Comment:
New changelog entries go to top of the file.

Reply:
Ah, sorry about that. 👍